docs/pages/admin-guides/management/guides/awsoidc-integration-rds.mdx

```diff
@@ -0,0 +1,289 @@
+a tunnel connection to your Teleport cluster.
+
+This is a serverless HA approach to deploying the Teleport database service.
+It does not require any allowed inbound traffic from the internet and supports
+[automatic agent upgrades](../../../upgrading/automatic-agent-updates.mdx).
+
+The Teleport database service instances use
+[dynamic database registration](../../../enroll-resources/database-access/guides/dynamic-registration.mdx)
+so that you can enroll or disenroll databases without restarting or modifying
+the deployment.
+The instances will match and proxy connections for all dynamic `db` resources in
+Teleport that have the following labels:
+
+```yaml
+account-id: "<AWS account ID>"
+region: "<AWS region>"
+vpc-id: "<AWS VPC ID>"
+```
+
+{/* TODO(gavin): add the deployment diagram here once it is finalized */}
+
+AWS resources created by the RDS enrollment wizard are tagged in AWS so that you
+can search and export them using the
+[AWS Resource Groups / Tag Editor](https://console.aws.amazon.com/resource-groups/tag-editor/find-resources).
+The following tags are applied:
+
+```code
+teleport.dev/cluster      <Var name="cluster-name"/>
+teleport.dev/origin       integration_awsoidc
+teleport.dev/integration  <Var name="my-integration"/>
+```
+
+You can also search for AWS resources created by the wizard using the `aws` cli:
+
+```code
+$ aws resourcegroupstaggingapi get-resources \
+--region <Var name="us-west-1"/> \
+--tag-filters \
+Key=teleport.dev/cluster,Values=<Var name="cluster-name"/> \
+Key=teleport.dev/integration,Values=<Var name="my-integration"/> \
+Key=teleport.dev/origin,Values=integration_awsoidc
+```
+
+## How to find the ECS deployment
+
+Navigate to the ECS service in AWS: https://console.aws.amazon.com/ecs/v2/clusters
+Make sure you select the same AWS region that you selected at the start of the
+RDS enrollment wizard.
+
+The ECS cluster that the wizard created will be named after your Teleport
+cluster except with dots `.` replaced with underscores `_` and the additional
+suffix `-teleport`.
+For example if your Teleport cluster is named
+`example.teleport.sh`, then the corresponding ECS cluster will be named
+`example_teleport_sh-teleport`.
+
+Click on your ECS cluster.
+
+![Screenshot of ECS clusters overview](../../../../img/database-access/enrollment-wizard/ecs-clusters-overview@2x.png)
+
+An ECS cluster is composed of one or more ECS services.
+Click on the service named after the VPC you chose in the enrollment wizard -
+the VPC ID is used as the suffix of the corresponding ECS service name, e.g
+`database-service-vpc-123456789abcdefgh`.
+
+![Screenshot of ECS services overview](../../../../img/database-access/enrollment-wizard/ecs-services-overview@2x.png)
+
+In the ECS service overview page you should click on the "Tasks" panel to see
+the deployment tasks.
+There should be two tasks either running or pending.
+If you don't see any tasks, check the task status filter:
+
+## Troubleshooting
```

Comment:
not sure what's the best place but could we add a section on how to find teleport logs through cloud watch?

Reply:
Good idea. I'll add info about how to find logs once a task is actually running

Reply:
added info about finding logs in [2b31627](https://github.com/gravitational/teleport/pull/48652/commits/2b316271e5fd9496db4c106da93de4e45005f1c6)